library/src/main/scala/scala/scalajs/js/BigInt.scala

```diff
@@ -0,0 +1,188 @@
+package scala.scalajs.js
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSGlobal
+
+/**
+ * A built-in object that provides a way to represent whole numbers larger than
```

Comment:
Please follow our coding style for Scaladoc comments, as explained at https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#scaladoc